retailer-oas/retailer.yaml

```diff
@@ -120,225 +96,310 @@ paths:
                 - clientSecret
                 - token
       responses:
-        '200':
+        200:
           description: OK
 
+  # Service availability
+  /service-availability:
```

Comment:
Please review this endpoint. I made some changes according to today's discussion (included service type and new response format).